docs/rpi.md

```diff
@@ -1,58 +1,41 @@
-#Raspberry Pi
+# Raspberry Pi Setup 
```

Comment:
This is a far clearer title for what this documentation explains, but the main README.md needs to be updated to reflect this title.